features/vat-replacement/run.js

```diff
@@ -0,0 +1,239 @@
+                    country: taxRateDraft.country
+                }
+            }
+        ]
+    }
+    return updateJsonObj
+
+}
+
+async function processTaxRate({ctpClient, validGermanTaxRateList, taxRateIdToTaxCategoryMap, taxRateType, isDryRun}) {
+    let oldTaxRate =  0
+    let newTaxRate =  0
+    if (taxRateType === vatConstant.TAX_RATE_TYPE_STANDARD) {
+        oldTaxRate = vatConstant.TAX_RATE_STANDARD_OLD
+        newTaxRate = vatConstant.TAX_RATE_STANDARD_NEW
+    } else {
+        oldTaxRate = vatConstant.TAX_RATE_REDUCED_OLD
+        newTaxRate = vatConstant.TAX_RATE_REDUCED_NEW
+    }
+
+    let taxRateDraftList =  validGermanTaxRateList.filter(rate => rate.amount === oldTaxRate)
+    if (taxRateDraftList.length>=1) {
+        let clonedTaxRateDraftList = JSON.parse(JSON.stringify(taxRateDraftList))
+        for (const taxRateDraft of clonedTaxRateDraftList) {
+            taxRateDraft.amount  = newTaxRate
+            const updateJsonObj = await getUpdateJsonObj(taxRateDraft, taxRateIdToTaxCategoryMap)
+            if (!isDryRun) {
+                console.log('Start to replace tax rate ... ')
+                await replaceTaxRate(ctpClient, taxRateIdToTaxCategoryMap.get(taxRateDraft.id).id, updateJsonObj)
+                console.log('Update finished')
+            } else {
+                console.log('Current tax rate would be replaced as below : ')
+                console.log(JSON.stringify(updateJsonObj))
+            }
+        }
+
+    }
+}
+
+async function printPreviewModeWarning() {
+    console.log('********************************************************************')
+    console.log('The script has been run in preview mode. To update the tax rates, ' + '\n' +
+        'you can input argument "-update" in CLI to run this script.')
+    console.log('Please make sure to resolve all tax categories and rates warnings ' + '\n' +
+        'before running the script in update mode.')
+    console.log('********************************************************************')
+}
+
+async function initConfigOptions() {
+    const args = process.argv.slice(2);
+    const configOptions = {
+        ctp : {
+            projectKey : process.env.PROJECT_KEY,
+            clientId : process.env.CLIENT_ID,
+            clientSecret : process.env.CLIENT_SECRET
+        },
+        dryRun: true
+    }
+
+    // Check environment variables
+    if (!configOptions.ctp.projectKey || !configOptions.ctp.clientId || !configOptions.ctp.clientSecret) {
+        console.error('Please set project key, client ID and client secret in environment variables')
+        return
+    }
+
+    // Check update mode / preview mode
+    if (args.length>0 && args[0] === '-update') {
+        configOptions.dryRun = false
+    }
+    return configOptions
+}
+(async function main() {
+    const configOptions = initConfigOptions()
+    const ctpClient = ctpUtils.setUpClient(configOptions)
+
+    // Retrieve tax category list in current project
```

Comment:
I don't think you need these comments.

Reply:
https://github.com/commercetools/commercetools-cli-scripts/pull/1/commits/a2de8303de7d55151a1508844e7b8709deb64eb8